CRM/Utils/Check/Component/Security.php

```diff
@@ -393,8 +393,16 @@ public function guessUrl($targetDir) {
     $filePathMarker = $this->getFilePathMarker();
     $config = CRM_Core_Config::singleton();
 
-    list ($heuristicBaseUrl, $ignore) = explode($filePathMarker, $config->imageUploadURL);
-    list ($ignore, $heuristicSuffix) = explode($filePathMarker, str_replace('\\', '/', $targetDir));
+    $res_arr = explode($filePathMarker, $config->imageUploadURL);
+    $heuristicBaseUrl = $res_arr[0];
```

Comment:
These two lines could be simplified as

    list ($heuristicBaseUrl) = explode($filePathMarker, $config->imageUploadURL);